nengo/learning_rules.py

```diff
@@ -7,12 +7,10 @@
 
 
 class ConnectionParam(NengoObjectParam):
```

Comment:
Ah yeah, I forgot about that. This'll get merged first probably and the work's done so no harm in changing it.